src/estimator/mean-squared-error.ts

```diff
@@ -0,0 +1,77 @@
+import { makeKernel } from '../utilities/kernel';
+import { IKernelRunShortcut } from 'gpu.js';
+
+/**
+ * 2D Mean Squared Error
+ */
+export function mse2d(
+  this: {
+    constants: { height: number; width: number; length: number };
+  },
+  errors: Array<[number, number]>
```

Comment:
Is this the linter? This should have been in the older commits. I believe the linter is running after the commit. Is this a known issue?

Reply:
Please pull master into HarshKhandeparkar:js-needs-types

Reply:
Done. This is still there though. Should it happen precommit?

Reply:
We run `npm run lint` as `pre-commit` hook. So if there is any error/warning in code it will be notified.

Reply:
Ohk. I used the VSCode git extension so I didnt get any errors.
If that hook makes any changes, doesn't it commit those changes?